go/state/gostate/archive_state.go

```diff
@@ -14,12 +14,12 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/Fantom-foundation/Carmen/go/database/mpt"
+	mptio "github.com/Fantom-foundation/Carmen/go/database/mpt/io"
```

Comment:
nit: this looks like an unrelated change in for this PR; maybe this file change should be removed

Reply:
oh, right, was updated by my IDE